### DISCUSSION TITLE
Release 3.6.1

### DISCUSSION THREAD

**Discussion by blapie** [shibatch/sleef#538] [General]
Hello,

Following several bug fixes in SLEEF 3.6, as well as the need for a fix in the API, we are planning a patch release in the next month.

## Release Schedule

Feature Freeze: 31 May 2024 
Validation: 31 May 2024  - 07 Jun 2024 
Release Date: 10 Jun 2024

## Release Scope

The idea of this release is to have a release tag containing all recent fixes to 3.6 (in particular a fix in API)

### Current Changelog

* Shared libs + major cmake update #531, #535
* Complete support for RVV 
* Address sanitizing #464, #528, #533
* Docs update #519 , #529

### Still TODO

* Fix in API #534 
* Issue with gcc 12 is fixed but isn't a bug in SLEEF - novelty is official support and test gcc 14. #439, #483
* Add validation tests to GHA (tentative)

**Comment by luhenry** [comment]
Hi, great to see a new release (especially with RISC-V support!). Will https://github.com/shibatch/sleef/pull/536 and https://github.com/shibatch/sleef/pull/537 be part of it as well? It would be great to have these as these can be argued to be correctness fixes. They respectively allow to link multiple objects each with their reference to `Sleef_rempitabdp`, and fix the inlining of functions marked as `SLEEF_ALWAYS_INLINE`.

**Reply by blapie** [reply]
Sure, anything that is already on the master branch will make it to the patch release, that includes these 2 patches.

**Comment by blapie** [comment]
The patch release 3.6.1 is live, more details at https://github.com/shibatch/sleef/releases/tag/3.6.1
